_test/check-uuids.sh

```diff
@@ -1,11 +1,13 @@
-#!/bin/sh
+#!/usr/bin/env bash
+set -eo pipefail
 
 repo=$(cd "$(dirname "$0")/.." && pwd)
 
 # Check for invalid UUIDs.
 # can be removed once `configlet lint` gains this ability.
 # Check issue https://github.com/exercism/configlet/issues/99
-bad_uuid=$(jq --raw-output '.exercises | map(.uuid) | .[]' $repo/config.json | grep -vE '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$')
+
+bad_uuid=$(jq --raw-output '.exercises | [.concept[], .practice[]] | map(.uuid) | .[]' "$repo"/config.json | grep -vE '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$' || test 1)
```

Comment:
```suggestion
bad_uuid=$(jq --raw-output '.exercises | .concept[], .practice[] | .uuid' "$repo"/config.json | grep -vE '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$' || test 1)
```

Test this before merging. I'm pretty confident this is a simplification with identical behavior, but I am on my phone right now and haven't verified it.

Reply:
I checked it and both commands produce identical output, so let's use the simplification.

Reply:
Perfect; thanks! I was fumbling around with the `map()` syntax anyways :)